### PR TITLE
Binheap

### PR DESCRIPTION
**Pull request by jeremytedwards**

### CODE REVIEW

File: src/test_binheap.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from builtins import range
+
+import pytest
+
+
+TEST_ITEMS = [
+    [],
+    list(range(10)),
+    list(reversed(range(10))),
+    [1] * 5,
```
Comment (joelstanner) [inline src/test_binheap.py:11]: Consider adding a test for a case of only 1 item. That's a common edge case.
